src/Microsoft.AspNetCore.Blazor.Build/Cli/Commands/ResolveRuntimeDependenciesCommand.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Blazor.Build.Core;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.Blazor.Build.Cli.Commands
+{
+    class ResolveRuntimeDependenciesCommand
+    {
+        public static void Command(CommandLineApplication command)
+        {
+            var references = command.Option("--reference",
+                "Full path to the application dependencies that are not part of netstandard2.0",
```

Comment:
That's a bit overly specific about `netstandard2.0` (which will change, and arguably no dependency except `netstandard.dll` is part of `netstandard2.0` anyway). Can this just say `Full path to a referenced assembly file`?